src/api/environment.cc

```diff
@@ -354,12 +354,44 @@ Local<Context> NewContext(Isolate* isolate,
   auto context = Context::New(isolate, nullptr, object_template);
   if (context.IsEmpty()) return context;
 
+  InitializeContextRuntime(context);
```

Comment:
It looks more reasonable to do this later than `InitializeContext()` here since logically `InitializeContext` is supposed to give you a context that looks like the one created with `Context::FromSnapshot`